### ISSUE TITLE
Android Sdk Version 16 compatibility  (vectorDrawables.useSupportLibrary)

### ISSUE THREAD

**Issue by RenatoMartinsFerreira** [react-native-picker/picker#319]
Hello! 
 Using react-native-picker/picker on a Android project with `minSdkVersion = 16` provoked the following error
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-picker_picker:packageDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Error while processing /Users/******/*******/*******/*********/node_modules/@react-native-picker/picker/android/src/main/res/drawable/ic_dropdown.xml : Can't process attribute android:fillColor="@android:color/black": references to other resources are not supported by build-time PNG generation.
     File was preprocessed as vector drawable support was added in Android 5.0 (API level 21)
     See http://developer.android.com/tools/help/vector-asset-studio.html for details.
```

 Searching for a solution my team and I found a section in the [Android documentation talking about vector compatibility](https://developer.android.com/guide/topics/graphics/vector-drawable-resources#vector-drawables-backward-solution) and worked!
 
  ```
    //For Gradle Plugin 2.0+
     android {
       defaultConfig {
         vectorDrawables.useSupportLibrary = true
        }
     }
```

[Here is a Pull Request ](https://github.com/react-native-picker/picker/pull/318)solving this issue, if you have any comment, please, let's start a talk! ;D

Thanks!

**Comment by lai-caju** [comment]
Oh God! I really need this PR, is there any other temporary solution for this, until it's merged?

**Comment by RenatoMartinsFerreira** [comment]
> Oh God! I really need this PR, is there any other temporary solution for this, until it's merged?

 Yes certainly! there are many, here are three:

1.  Reference the git repository on package.json
 ``` 
  "dependencies": {
...
    "@react-native-picker/picker": "git+https://github.com/RenatoMartinsFerreira/picker.git#issue-android-sdk-16-support",
...
}
  ```
2.  Download the lib to a project directory, make necessary changes and reference it on package.json:
  ``` 
"dependencies": {
...
    "@react-native-picker/picker": "file:./your-edited-libs/react-native-picker/picker",
...
} 
```
**Im not sure if it can increase the APP size**

3. Depending on your company, as the lib is on MIT License, you can modify it and add to your private dependency manager.


I hope it was helpful, any doubt just ask!

**Comment by Naturalclar** [comment]
@RenatoMartinsFerreira thanks for raising the issue!
The issue should be resolved via #321 :tada:
Please use the latest version of the picker.
